models/schemas/reviews.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require('mongoose');
+
+const reviewSchema = new mongoose.Schema({
```

Comment:
Could you also add in a gameID field, so you can quickly look up a game, instead/as well as a title

Reply:
Do you mean with an ID I assign?

Reply:
Well, or link it to a document using the `_id` property?

Reply:
Okay, yeah, so it doesn't matter if the same game ID is referenced across multiple reviews? I think you're right, that's the way to go.